### PR TITLE
adding new features to the compiler

### PR DESCRIPTION
**Pull request by bryan-era24**

### CODE REVIEW

File: src/frontend/Main.form [hunk 1]
```diff
@@ -98,7 +98,7 @@
   </NonVisualComponents>
   <Properties>
     <Property name="defaultCloseOperation" type="int" value="0"/>
-    <Property name="title" type="java.lang.String" value="Compilador Gustavo Quintero"/>
```
Comment (GustavoQuintero99) [inline src/frontend/Main.form:101]: ???

**Review by GustavoQuintero99 — APPROVED** [review]
LGTM

**Comment by GustavoQuintero99** [comment]
thanks for the change

File: src/frontend/Main.form (hunk 1 above)
Comment (Miltonmzr) [reply src/frontend/Main.form:101]: Bro that's amazing 😎

**Pull request merged**